src/string_id.h

```diff
@@ -127,6 +127,15 @@ class string_id
          * Returns the actual object this id refers to. May show a debug message if the id is invalid.
          */
         const T &obj() const;
+
+        const T &operator*() const {
+            return obj();
+
+        }
+        const T *operator->() const {
+            return &obj();
+        }
```

Comment:
It's a lot less ugly than `foo.obj().quux.obj()`

@BevapDin wrote that class and didn't have any objections:

> A bit inconsistent: why use obj() here instead of the new operator->?

More things should be `string_id` and providing C++ operators as opposed to our own custom functions improves readability. There's no additional risk over just calling `obj()`.